.eslintrc

```diff
@@ -18,12 +19,19 @@
     "standard",
     "standard-jsx",
     "plugin:jsx-a11y/recommended",
-    "plugin:react/recommended"
+    "plugin:react/recommended",
+    "plugin:import/recommended",
+    "plugin:n/recommended"
   ],
   "rules": {
     "jsx-quotes": [2, "prefer-double"],
     "jsx-a11y/no-onchange": "off",
-    "react/prop-types": "off"
+    "react/prop-types": "off",
+    "n/no-missing-require": "off",
+    "n/no-unsupported-features/es-syntax": "off",
+    "n/no-missing-import": "off",
```

Comment:
linting regarding imports is done better by the other package we use (e.g. `eslint-plugin-import`). So disabling these rules.